tests/Application/.env

```diff
@@ -35,12 +35,3 @@ SYLIUS_MESSENGER_TRANSPORT_CATALOG_PROMOTION_REMOVAL_FAILED_DSN=doctrine://defau
 ###> payum/payum ###
 PAYUM_CYPHER_KEY=<edit_me>
 ###< payum/payum ###
-
-###> commerce-weavers/sylius-tpay-plugin ###
-TPAY_CLIENT_ID=<edit_me>
-TPAY_CLIENT_SECRET=<edit_me>
-TPAY_CARDS_API=<edit_me>
-TPAY_NOTIFICATION_SECURITY_CODE=<edit_me>
-TPAY_MERCHANT_ID=<edit_me>
-TPAY_GOOGLE_MERCHANT_ID=<edit_me>
-###< commerce-weavers/sylius-tpay-plugin ###
```

Comment:
These values are not intended to be used in the end-app, so I removed them to not confuse they are required to be set.

Reply:
but then we need to update installation wiki